openbb_terminal/cryptocurrency/onchain/blockchain_view.py

```diff
@@ -129,3 +132,57 @@ def display_btc_confirmed_transactions(
         "btcct",
         df,
     )
+
+
+@log_start_end(log=logger)
+def display_btc_single_block(
+    blockhash: str,
+    export: str = "",
+) -> None:
+    """Returns BTC block data. [Source: https://api.blockchain.info/]
+    Parameters
+    ----------
+    blockhash : str
+        Hash of the block you are looking for.
+    export : str
+        Export dataframe data to csv,json,xlsx file
+    """
+
+    df = blockchain_model.get_btc_single_block(blockhash)
+    df.rename(index={0: "Values"}, inplace=True)
+    df_data = df.copy()
+
+    df_essentials = df[[
```

Comment:
@DidierRLopes should this transformation be done here or in the model?

Reply:
Done